source/_docs/unsupported-modules-plugins.md

```diff
@@ -203,83 +218,75 @@ or to a specific directory:
 ```
 define('AUTOPTIMIZE_CACHE_CHILD_DIR','/uploads/autoptimize/');
 ```
-For additional details, see the [Autoptimize's FAQ](https://wordpress.org/plugins/autoptimize/faq). An alternative solution is to [create a symblic link](/docs/assuming-write-access/#create-a-symbolic-link).
+For additional details, see the [Autoptimize FAQ](https://wordpress.org/plugins/autoptimize/faq). An alternative solution is to [create a symbolic link](/docs/assuming-write-access/#create-a-symbolic-link).
 
-### Contact Form 7
+### [Contact Form 7](https://wordpress.org/plugins/contact-form-7/)
 **Issue**: This plugin utilizes a static value, `$_SERVER['SERVER_NAME']`, instead of `$_SERVER['HTTP_HOST']` which is generated dynamically.
 
 **Solution**: Add the following to `wp-config.php`: `$_SERVER['SERVER_NAME'] = $_SERVER['HTTP_HOST'];`
 
 For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_name-and-server_port/).
 <hr>
 
-### Caching Plugins (e.g. Batcache, W3 Total Cache, or WP Super Cache)
+### Caching Plugins (e.g. [Batcache](https://wordpress.org/plugins/batcache/), [W3 Total Cache](https://wordpress.org/plugins/w3-total-cache/), or [WP Super Cache](https://wordpress.org/plugins/wp-super-cache/))
 **Issue**: Conflicts with Varnish.
 
 **Solution**: See [Caching: Advanced Topics](/docs/caching-advanced-topics/) for details on how to bypass Varnish.
 <hr>
 
-### Instashow
+### [Instashow](https://elfsight.com/instagram-feed-instashow/)
 **Issue**: The Instashow plugin relies on query parameters that are not compatible with Pantheon's Edge Cache. See [PANTHEON_STRIPPED](https://pantheon.io/docs/pantheon_stripped/) for more information. This inhibits the ability to set the authorization token required to make the plugin function.
 <hr>
 
-### Monarch Social Sharing
+### [Monarch Social Sharing](https://www.elegantthemes.com/plugins/monarch/)
 **Issue**: Seems to break WP-CLI, which is used by many of our workflows (clone, clear cache).
 <hr>
 
-### Revive Old Post
+### [Revive Old Post](https://wordpress.org/plugins/tweet-old-post/)
 **Issue**: Revive Old Post does not set a proper callback via OAuth and the Twitter module.  It attempts to use ["SERVER_NAME"] instead of the recommended ["HTTP_HOST"]. See [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_name-and-server_port/).
 
 <hr>
 
-### Sucuri Scanner
+### [Sucuri Security](https://wordpress.org/plugins/sucuri-scanner/)
 **Issue**: Sucuri Scanner enforces absolute file paths for the data storage path and the container ID of the file path will change with routine platform maintenance.
 <hr>
 
-### Timthumb
-**Issue**: TimThumb is no longer supported or maintained. See [https://code.google.com/p/timthumb/](https://code.google.com/p/timthumb/).
+### [Timthumb](https://code.google.com/p/timthumb/)
```

Comment:
Not sure this makes sense in the WordPress section, since it doesn't reference plugins or a specific CMS. Maybe it can go at the end?